src/java/main/org/apache/zookeeper/server/NettyServerCnxn.java

```diff
@@ -1,856 +0,0 @@
-        @Override
-        public void commandRun() {
-            if (zkServer == null) {
-                pw.println(ZK_NOT_SERVING);
-            } else {
-                zkServer.dumpConf(pw);
-            }
-        }
-    }
-    
-    private class StatResetCommand extends CommandThread {
-        public StatResetCommand(PrintWriter pw) {
-            super(pw);
-        }
-        
-        @Override
-        public void commandRun() {
-            if (zkServer == null) {
-                pw.println(ZK_NOT_SERVING);
-            }
-            else { 
-                zkServer.serverStats().reset();
-                pw.println("Server stats reset.");
-            }
-        }
-    }
-    
-    private class CnxnStatResetCommand extends CommandThread {
-        public CnxnStatResetCommand(PrintWriter pw) {
-            super(pw);
-        }
-        
-        @Override
-        public void commandRun() {
-            if (zkServer == null) {
-                pw.println(ZK_NOT_SERVING);
-            } else {
-                synchronized(factory.cnxns){
-                    for(ServerCnxn c : factory.cnxns){
-                        c.resetStats();
-                    }
-                }
-                pw.println("Connection stats reset.");
-            }
-        }
-    }
-
-    private class DumpCommand extends CommandThread {
-        public DumpCommand(PrintWriter pw) {
-            super(pw);
-        }
-        
-        @Override
-        public void commandRun() {
-            if (zkServer == null) {
-                pw.println(ZK_NOT_SERVING);
-            }
-            else {
-                pw.println("SessionTracker dump:");
-                zkServer.sessionTracker.dumpSessions(pw);
-                pw.println("ephemeral nodes dump:");
-                zkServer.dumpEphemerals(pw);
-            }
-        }
-    }
-    
-    private class StatCommand extends CommandThread {
-        int len;
-        public StatCommand(PrintWriter pw, int len) {
-            super(pw);
-            this.len = len;
-        }
-        
-        @Override
-        public void commandRun() {
-            if (zkServer == null) {
-                pw.println(ZK_NOT_SERVING);
-            }
-            else {   
-                pw.print("Zookeeper version: ");
-                pw.println(Version.getFullVersion());
-                if (zkServer instanceof ReadOnlyZooKeeperServer) {
-                    pw.println("READ-ONLY mode; serving only " +
-                               "read-only clients");
-                }
-                if (len == statCmd) {
-                    LOG.info("Stat command output");
-                    pw.println("Clients:");
-                    // clone should be faster than iteration
-                    // ie give up the cnxns lock faster
-                    HashSet<ServerCnxn> cnxns;
-                    synchronized(factory.cnxns){
-                        cnxns = new HashSet<ServerCnxn>(factory.cnxns);
```

Comment:
### ✅ No longer an issue: Bumpy Road Ahead
NettyServerCnxn.StatCommand.commandRun is no longer above the threshold for logical blocks with deeply nested code